src/css/controls/imports/shortcuts-tooltip.less

```diff
@@ -77,8 +73,11 @@
             }
         }
 
-        li {
-            line-height: 34px;
+        .jw-shortcuts-tooltip-keys,
+        .jw-shortcuts-tooltip-descriptions {
+            div {
```

Comment:
it's necessary because I want to style all divs that are children of those two divs. Global selector performance is eh, and adding a class to each child element and styling that class is more work for the browser (same number of elements receiving styles, but adding a class check). I think given the required changes this is the best solution.